package.json

```diff
@@ -27,6 +27,7 @@
     "emojilib": "^2.4.0",
     "eth-phishing-detect": "^1.1.13",
     "functional-red-black-tree": "^1.0.1",
+    "hermes-engine": "0.5.2-rc1",
```

Comment:
**TODO** - remove `hermes-engine 0.5.2-rc1` once react-native is upgraded to 0.64+ _(will create separate issue for this)_
https://github.com/wix/react-native-notifications/issues/455#issuecomment-803413032